stock_removal_location_by_priority/models/stock_quant.py

```diff
@@ -14,7 +14,8 @@ class StockQuant(models.Model):
     )
 
     @api.model
-    def _get_removal_strategy_order(self, removal_strategy=None):
+    def _get_removal_strategy_order(self, removal_strategy):
+        self.flush()
```

Comment:
Could you clarify why this is needed?

Reply:
I think this is because computed (or related) fields are not automatically recomputed.

@ps-tubtim If this is needed for tests purpose, I suggest to use it specifically in tests.

Reply:
@rousseldenis Thank you for suggestion. Edited.

Reply:
@rousseldenis But if you look at the Odoo code, it is not there, so that made me think that it was not needed there and the only reason was to make test work (what was the case regarding the latest update)

@ps-tubtim Thanks for the fix!

Reply:
> @rousseldenis But if you look at the Odoo code, it is not there, so that made me think that it was not needed there and the only reason was to make test work (what was the case regarding the latest update)

Your feeling was good.

flush() will commit everything in cache, so only useful in tests (see tests code in Odoo). A contrario as I said, it is only used when calling load() as we want to have fresh data.